nltk/classify/senna.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, senna_path, operations, encoding="utf-8"):
                 self._path = path.normpath(environ["SENNA"]) + sep
                 exe_file_2 = self.executable(self._path)
                 if not path.isfile(exe_file_2):
-                    raise OSError(
+                    raise LookupError(
```

Comment:
I believe that `LookupError` is more logical here. We also use `LookupError` in other cases where looking up a certain resource fails.